sdks/java/core/src/main/java/org/apache/beam/sdk/schemas/Schema.java

```diff
@@ -252,7 +264,9 @@ public FieldType type() {
     @AutoValue.Builder
     abstract static class Builder {
       abstract Builder setTypeName(TypeName typeName);
-      abstract Builder setComponentType(@Nullable FieldType componentType);
+      abstract Builder setCollectionType(@Nullable FieldType collectionType);
```

Comment:
<!--new_thread; commit:4ec9e60e06584a642db426cf6cf5ccd57cb62337; resolved:0-->
nitpick here: collectionType reads weird to me, as it seems like it's the type of collection (e.g. array, list, tree, etc.) instead of the type of the component elements of the collection.

Reply:
I didn\t mind componentType, but if you want to include collection in there then maybe CollectionElementType or CollectionValueType?

Reply:
ok, LGTM from me once that change is made.

Reply:
updated, please have a look when you've time.